src/js/themes/base.d.ts

```diff
@@ -81,10 +81,12 @@ type Colors = typeof colors & {
 
 interface ButtonKindType {
   background?: BackgroundType;
-  border?: {
-    color?: ColorType;
-    width?: string;
-  } | boolean;
+  border?:
```

Comment:
Maybe change linter back to how it is

Reply:
This add extra pipe before

Reply:
when i commit it auto lints it for me. is there a config im missing?

Reply:
hmm maybe that got changed ? @ShimiSun is this the right linting ?

Reply:
We recently added linter & prettier for typescript files. So I think it might be a side effect, we'll need to take a look whether that is the default behavior that is a best practice.

Reply:
do i need a new config file, or to adjust the current config to account for this?

Reply:
It will probably be a separate task from this PR, leave it be for now if its complaint with lint.